### DISCUSSION TITLE
Weird default trailingSlash behaviour

### DISCUSSION THREAD

**Discussion by franciskafieh** [vercel/vercel#8443] [Ideas]
The [vercel.json docs](https://vercel.com/docs/project-configuration#project-configuration/trailing-slash) explain that by default, `/page` and `/page/` will serve the same content without redirecting. They go on to explain how it's bad for SEO. Do you think it would be better to set the default to `true` or `false` instead? The problem being which one to set it to I guess. It just doesn't make sense imo to knowingly provide a harmful default. What do you all think?

**Comment by amyegan** [comment]
This is a great question @franciskafieh! I believe that Vercel's default `undefined` behavior of `trailingSlash` is meant to accommodate many possible framework options and allow granular control over redirects for apps that require it. But this is not my area of expertise, so I'll ask the Engineering team and let you know if they have anything else to say about it.

**Reply by franciskafieh** [reply]
Thank you! Maybe when setting up a project, urge the user to change this option? This way, the user can pick whichever option will work best for their framework. Without reading the docs, many users may be hit by this SEO impact without knowing.

**Reply by amyegan** [reply]
I'm adding it to our requested features list. Thanks for bringing this up!
